specification/columns/availabilityzone.md

```diff
@@ -1,27 +1,27 @@
 # Availability Zone
 
-[Availability Zone](#glossary:availability-zone) is a provider assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
+An [*availability zone*](#glossary:availability-zone) is a provider-assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
 
-The AvailabilityZone column SHOULD be present in the billing data. This column MUST be of type String and MAY contain null values.
+The AvailabilityZone column MUST be present in the billing data when the provider supports deploying resources or services within an *availability zone*. This column MUST be of type String and MAY contain null values when a charge is not specific to an *availability zone*.
```

Comment:
I'm not okay with changing AvailabilityZone from SHOULD to MUST. Can we revert it back?

Reply:
follow up discussion raised in issue #367